lib/phases/reboot.js

```diff
@@ -97,6 +99,7 @@ Reboot.prototype.capture = function() {
    */
   this.dispatcher.on(Phase.PERFORMANCEENTRY, function(entry) {
     var name = entry.name;
+    var entryOut = entry;
```

Comment:
error Assignment to function parameter 'entry' no-param-reassign (called out on line 120 what originally was entry.entryPoint = entryPoint.